tutorials/01-DataJoint Basics.ipynb

```diff
@@ -724,16 +829,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Querying data"
+    "### Querying data"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Often times, you don't want all data but rather work with **a subset of entities** matching specific criteria. Rather than fetching the whole data and writing your own parser, it is far more efficient to narrow your data to the subset before fetching.\n",
+    "Oftentimes, you want only some of the data but rather work with **a subset of entities** matching specific criteria. Rather than fetching the whole data and writing your parser, narrowing your data to the subset before fetching is far more efficient.\n",
```

Comment:
```suggestion
    "Oftentimes, you don't need to use all of the data but rather work with **a subset of entities** matching specific criteria. Rather than fetching the whole data and writing your parser, narrowing your data to the subset before fetching is far more efficient.\n",
```